pytorch_lightning/plugins/training_type/deepspeed.py

```diff
@@ -260,12 +259,6 @@ def __init__(
             load_full_weights: True when loading a single checkpoint file containing the model state dict
                 when using ZeRO Stage 3. This differs from the DeepSpeed checkpoint which contains shards
                 per worker.
-
-            partition_module: When True, partitions the ``LightningModule`` across devices when using ZeRO Stage 3.
```

Comment:
I am curious, why removing `partition_module`? It still seems useful in training mode.

Reply:
Check the related issue: https://github.com/PyTorchLightning/pytorch-lightning/issues/10510